Sources/TSCclibc/include/process.h

```diff
@@ -0,0 +1,11 @@
+#if defined(__linux__)
+
+#include <spawn.h>
+
+// Wrapper method for posix_spawn_file_actions_addchdir_np that fails on Linux versions that do not have this method available.
+int SPM_posix_spawn_file_actions_addchdir_np(posix_spawn_file_actions_t *restrict file_actions, const char *restrict path);
+
+// Runtime check for the availability of posix_spawn_file_actions_addchdir_np. Returns 0 if the method is available, -1 if not.
+int SPM_posix_spawn_file_actions_addchdir_np_supported();
```

Comment:
The name ending in _supported seems to imply this should return a `bool`, otherwise I'd expect something more like `SPM_posix_spawn_file_actions_addchdir_np_support_status` which could return 0, ENOSYS, etc.

Reply:
changed to bool instead